.github/workflows/ci-pre-land.yml

```diff
@@ -281,9 +281,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Ruby 2.6
-        uses: actions/setup-ruby@v1
+        uses: ruby/setup-ruby@v1
```

Comment:
@vgao1996 heads-up, I needed to change this too. Apparently the old action has been deprecated